docs/operate/mobility/move-arm/frame-how-to.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Configure your frame system"
+linkTitle: "Configure your frame system"
+weight: 20
+type: "docs"
+layout: "docs"
+description: "Decide on and configure the frames of your arm, its workspace, and other components."
+---
+
+Frames can be confusing.
+This guide will help you decide on a coordinate system for your workspace, and then determine how the frames of your arm and other components relate to that workspace.
+
+Once you have configured frames for all your components, Viam will keep track of the positions and orientations as your robot moves, so you can plan motion in terms of a consistent coordinate system.
+
+For reference information, see [the frame system](/operate/reference/services/frame-system/).
+
+## Determine the world frame
+
+The world reference frame is the fixed coordinate system that serves as the reference point for the other frames in your robotic system.
+
+You choose the world frame, in whatever way is convenient for your application.
+It generally makes sense to define the world frame's location as a point in your space that does not move and is easy to measure from.
+
+For example, if you are using a robot arm mounted to a table, it can be convenient to decide that the world frame origin is at the base of the arm, or you can define the world frame origin as one corner of the table that the arm is mounted to.
```

Comment:
```suggestion
For example, if you are using a robot arm mounted to a table, it can be convenient to decide that the world frame origin is at the base of the arm, or you can decide that the world frame origin is one corner of the table that the arm is mounted to.
```